package.json

```diff
@@ -21,14 +22,17 @@
     "*.{js,jsx,ts,tsx}": [
       "cross-env NODE_ENV=development eslint --cache"
     ],
-    "*.json,.{eslintrc,prettierrc}": [
-      "prettier --ignore-path .eslintignore --parser json --write"
+    "*.json": [
+      "prettier --write"
+    ],
+    ".{eslintrc,prettierrc}": [
+      "prettier --write"
     ],
     "*.{css,scss}": [
-      "prettier --ignore-path .eslintignore --single-quote --write"
+      "prettier --write"
     ],
     "*.{html,md,yml}": [
-      "prettier --ignore-path .eslintignore --single-quote --write"
+      "prettier --write"
     ]
```

Comment:
This is the main change.
- Instead of passing `--ignore-path`, I added a `.pretterignore` file symlinked to `.eslintignore`, so that this path will always be used even if we run the command without options.
- Instead of passing `--single-quote`, I added to `.prettierrc` so that this will always be used even if we run the command without options.